### ISSUE TITLE
Unable to authorize apple music account

### ISSUE THREAD

**Issue by MekoScreams** [ciderapp/Cider-2#1266]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [x] I have confirmed that no [support FAQ](https://cider.sh/docs/client/faqs) resolves my issue.
- [x] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [x] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### How did you download Cider?

itch.io

### App Information

{"clientInfo":{"client":"dotnet","env":"production","arch":"AMD64","platform":"win32","platformFriendly":"Windows","build":"desktop","clientVersion":"300","capi":"d4f32ed4-6c4b-4b1c-a96c-101e6a5ed308","subversion":"0"},"buildInfo":{"version":"3.0.9","commit":"92de20183d99a02ca7cdac5e6cfaa4ddfdbd4a51","branch":"main","buildDate":"1755828050374","changelogUrl":"3.0.0-release"}}

### Operating System

Windows

### OS Version

Windows 11

### How would you describe your issue?

Accessibility

### Commit Hash

92de20183d99a02ca7cdac5e6cfaa4ddfdbd4a51

### Describe the Bug

I was already playing music on the Cider app until it would only play one song and stop. I tried signing in and it said unable to authorize apple music. I tried a couple more times after that and still gives me the same thing. I tried logging into my apple music app and it doesn't do that on there.

### Steps to Reproduce

I tried signing in again by clicking the sign into apple music button and proceeded with the log in steps and received the error failed to authorize apple music.

**Comment by Kraax** [comment]
i have the same issue on Arch Linux

**Comment by K3NOXOFFICIAL** [comment]
Same error on Windows 24H2 using Cider 3.1.6 (PTB 200)

**Comment by Zachmd14** [comment]
Same for me but I cannot connect on apple music web too whatever the browser. Icloud still work tho.

**Comment by tort-oise** [comment]
same error on windows 10 latest, same issue as zachmd14 as well

**Comment by kukyys** [comment]
Same for me on Arch Linux, it just stopped playing all of a sudden. Logging in thru apple acc does that error you said, and thru user token a cannot see any of my music and it wont load playlists

**Comment by Zachmd14** [comment]
Seems like an update came out since, it's working fine again for me.

**Comment by fk0u** [comment]
same issue

**Comment by panicthescaredycat** [comment]
Amazing support from the Cider team :)
